### DISCUSSION TITLE
Keep controls visible while scrubbing on mobile

### DISCUSSION THREAD

**Discussion by theazgra** [muxinc/media-chrome#686] [Proposals]
Hi,

currently while scrubbing on mobile, the controls with timeline dissappear. This is rather strange to us.

Would it be possible to add option to keep controls visible while scrubbing?

I have originally opened bug report in Mux Player repository: https://github.com/muxinc/elements/issues/714

**Comment by jakubslambor** [comment]
+1

**Comment by luwes** [comment]
Closing discussion in favor of https://github.com/muxinc/media-chrome/issues/751
